universe/auto_syncer.go

```diff
@@ -76,7 +76,7 @@ type FederationPushReq struct {
 // FederationIssuanceBatchPushReq is used to push out a batch of new issuance
 // events to all or some members of the federation.
 type FederationIssuanceBatchPushReq struct {
-	IssuanceBatch []*IssuanceItem
+	IssuanceBatch []*Item
```

Comment:
Do we want to rename this (and the struct name) as well? Or is this actually just for issuances?

Reply:
Yes, I've added more commits to address this.